examples/getting_started/run.py

```diff
@@ -12,7 +12,96 @@
+        control_dead_zone       = self._config.OPENBOT_ENV.CONTROL_DEAD_ZONE       # Absolute duty cycle (in the ACTION_SCALE range) below which a command does not produce any torque on the vehicle
+        motor_torque_max        = self._config.OPENBOT_ENV.MOTOR_TORQUE_MAX        # Motor maximal torque [N.m]
+        electrical_resistance   = self._config.OPENBOT_ENV.ELECTRICAL_RESISTANCE   # Motor winding electrical resistance [Ohms]
+
+        # Speed multiplier defined in the OpenBot to map a [-1,1] action to a suitable command to 
+        # be processed by the low-level microcontroller. For more details, feel free to check the
+        # "speedMultiplier" command in the OpenBot code.
+        # https://github.com/isl-org/OpenBot/blob/master/android/app/src/main/java/org/openbot/vehicle/Vehicle.java#L375
+        action_scale = self._config.OPENBOT_ENV.ACTION_SCALE
+
+        # Acquire the ground truth motor and wheel velocity for motor counter-electromotive force
+        # computation purposes (or alternatively friction computation purposes).
+
+        # The ground truth velocity of the robot wheels in [rad/s]
+        wheel_rotation_speeds = self._wheel_rotation_speeds
+
+        # The ground truth rotation speed of the motors in [rad/s]
+        motor_speeds = gear_ratio * wheel_rotation_speeds
+
+        # Compute the counter electromotive force using the motor torque constant
+        counter_electromotive_forces = motor_torque_constant * motor_speeds # Expressed in [V]
+
+        # The electrical current allowed to circulate in the motor is the result of the
+        # difference between the applied voltage and the counter electromotive force
+        motor_winding_currents = ((battery_voltage * duty_cycles) - counter_electromotive_forces) / electrical_resistance # Expressed in [A]
+
+        # The torque is then obtained using the torque coefficient of the motor in [N.m]
+        drive_torques = motor_torque_constant * motor_winding_currents
+
+        # Motor torque is saturated to match the motor limits
+        drive_torques = np.clip(drive_torques, -motor_torque_max, motor_torque_max)
+
+        # The torque applied to the robot wheels is finally computed accounting for the gear ratio
+        wheel_torques = gear_ratio * drive_torques
+
+        # Control dead zone at near-zero speed. This is a simplified but reliable way to deal with
+        # the friction behavior observed on the real vehicle in the low-speed/low-duty-cycle regime.
+        # TODO: get value from the config system
+        wheel_torques[np.logical_and(abs(motor_speeds) < 1e-5, abs(duty_cycles) <= control_dead_zone / action_scale)] = 0.0
+
+        spear.log("    wheel_torques: ", wheel_torques)
+
+        # formulate action before sending it to the simulator
+        action_super = {"set_drive_torques": drive_torques, "set_brake_torques": np.zeros(shape=(4), dtype=np.float64)}
```

Comment:
Do you need `set_brake_torques` for anything? I don't think this should be part of `ACTION_COMPONENTS` if you're not using it for anything in the Python code.